main.nf

```diff
@@ -808,6 +819,8 @@ Channel.from(summary.collect{ [it.key, it.value] })
     """.stripIndent() }
     .set { ch_workflow_summary }
 
+log.info "Schaffa, Schaffa, Genome Baua!"
```

Comment:
?

Reply:
This is an easter egg we had in EAGERv1 and James wanted to make this available in the new version too as people got used to it ;-) Swabian German for "Work, Work, Build a genome!" (+-, hard to 100% transcribe)

Reply:
@apeltzer @MaxUlysse https://www.youtube.com/watch?v=831QzzTuCjA

Keep the easter egg please :)